MAINTAINING.md

```diff
@@ -0,0 +1,127 @@
+# RIOT Maintainer Guidelines
+
+This list presents a series of guidelines for maintainers, including technical
+and non-technical. The list is not exhaustive, and represents a baseline on
+things that should be ensured for contributions.
+
+Notes:
+-   Any of the items in this list might be skipped, if clearly and logically
+    articulated reasons are presented.
+-   The order of the steps is meant to maximize efficiency and minimize
+    overhead, with the philosophy that failing in step n makes steps n+x
+    obsolete. However, this efficiency can depend on the quality of the
+    submission itself. If the PR (Pull Request) is clearly not in a reviewable
+    state (for example, due to poor code cleanliness or overall design), it
+    might be more efficient to give the contributor some broader comments for
+    improvement before further review.
+-   This is a working document and any changes, additions, or other discussions
+    are encouraged, via PRs raised against the document.
+
+The list of RIOT maintainers, and the areas of RIOT they each maintain, can be
+found [here](https://github.com/RIOT-OS/RIOT/wiki/Maintainers).
+
+
+## Technical guidelines
+
+### 1. - Review the fundamentals
+
+Before spending the time on an in-depth code review, it's important to assess
+the overall validity of the PR.
+
+1.  Does the reasoning for this PR make sense? \
+    Are the requirements and design goals clearly thought out and clearly
+    expressed? \
+    Is the problem that the PR intends to solve clearly stated?
+2.  Is the solution presented in the PR as simple as possible to satisfy the
+    requirements, but no simpler?
+3.  Is the solution well designed on a high level?
+4.  Do the concepts used by the PR make sense?
+5.  Does the PR break with existing concepts?
+6.  Is the structure of the PR itself valid?
+7.  Are there clear and adequate instructions on how to test the PR? \
+    This may or may not include implemented tests as part of the PR.
+8.  Does the code compile and run?
+9.  Does this PR respect the rights of previous authors, either through
+    retaining their commits or by retaining their copyrights in the boilerplate
+    headers?
+10. Is the PR a duplicate of another PR?
+
+
+### 2. - Review the design of the code
+
+The following list is not exhaustive and addresses the coding issues we have
+regularly seen in the past. In particular, check that the
+[Best Practices for RIOT Programming](https://github.com/RIOT-OS/RIOT/wiki/Best-Practice-for-RIOT-Programming)
```

Comment:
The line length problem with URLs could be solved by using references. E.g.

```markdown
In particular, check that the [Best Practices] for RIOT Programming are followed.

...
2. [Check memory usage][Comparing build sizes]
...

[Best Practices]: https://github.com/RIOT-OS/RIOT/wiki/Best-Practice-for-RIOT-Programming
[Comparing build sizes]: https://github.com/RIOT-OS/RIOT/wiki/Comparing-build-sizes
```

IMHO that would also make the text more readable in clear text.

Reply:
Done